mod_modular_vanilla/hooks/entity/tactical/actor.nut

```diff
@@ -358,10 +358,10 @@
 		{
 			local overflowDamage = _hitInfo.DamageArmor;
 
-			if (this.getBaseProperties().Armor[_hitInfo.BodyPart] != 0)
+			if ((this.getBaseProperties().Armor[_hitInfo.BodyPart] * p.ArmorMult[_hitInfo.BodyPart]) != 0)
```

Comment:
In Vanilla this `if` does not check ArmorMult.
But we want the following code also to be skipped if any skill or entity-definition would cause one of the body parts to have an ArmorMult of 0. One example would be a `headless_effect`, which would set the ArmotMult of the head piece to 0, so that the player is not confused by that armor bar, when it consists of natural armor